src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -9,6 +9,7 @@ import { CheckIcon, CloseIcon } from "@chakra-ui/icons";
 import ApplicationInfo from "../../../components/ApplicationInfo";
 import DocumentList from "../../../components/DocumentList";
 import { getApplicationDetails } from "../../../services/applicationService";
+import { getApplicationDetails } from "../../../services/applicationService";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getApplicationDetails'. Consider to delete it or rename it.

'getApplicationDetails' is defined here:


(lint/suspicious/noRedeclare)